apps/lib/app_params.c

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Where did this lib folder come from? Maybe this should be a seperate PR to start moving files to a lib folder?

Reply:
I've been thinking about exactly that.  I also have a desire to split apps.c into smaller units instead of being the current hodge podge of all kinds of things...

But, this is a start.